Sources/Misc/Concurrency/OperationDispatcher.swift

```diff
@@ -68,7 +68,5 @@ extension OperationDispatcher {
 
 }
 
-#if swift(<5.8)
-// `DispatchQueue` is not `Sendable` as of Swift 5.7, but this class performs no mutations.
+// `DispatchQueue` is not `Sendable` as of Swift 5.8, but this class performs no mutations.
```

Comment:
I forgot to explain this.
Until this PR, turns out we weren't relying on `OperationDispatcher` being `Sendable`. This broke on Swift 5.8, because we didn't have an `#else`.
I don't think they're ever going to make `DispatchQueue` `Sendable`, but our `@unchecked Sendable` is still valid, hence why I removed the conditional.